flake8_import_order/__about__.py

```diff
@@ -1,24 +0,0 @@
-from __future__ import absolute_import, division, print_function
```

Comment:
I don't think this should still be deleted, right?

Reply:
These `__future__` imports can be dropped now that Python 2 has been dropped.
* https://docs.python.org/3/library/__future__.html

`pyupgrade` will automate that process.